src/commands/commandui.cpp

```diff
@@ -172,6 +168,15 @@ int CommandUI::run(QStringList& tokens) {
     // If there is another instance, the execution terminates here.
     if (!EventListener::checkOtherInstances(
             I18nStrings::instance()->t(I18nStrings::ProductName))) {
+      // If we are given URL parameters, send them to the UI socket and exit.
```

Comment:
Consider extracting a method for this block and sharing with the code that executes when another instance of the app is not found.

Reply:
Never mind, the differences in the code blocks may not make it worth it.